src/cpp/server/server_cc.cc

```diff
@@ -1047,21 +976,14 @@ Server::~Server() {
       for (const auto& value : sync_req_mgrs_) {
         value->Shutdown();
       }
-      if (callback_cq_ != nullptr) {
-        callback_cq_->Shutdown();
+      if (callback_cq_.load() != nullptr) {
```

Comment:
there are two seq-cst loads here. Since it's in the dtor, I would make them both relaxed atomics.

Reply:
Thanks! Done.